### PR TITLE
Localised the variables from the form to each user.

### PR DESCRIPTION
**Pull request by BOluokun**
Instead of using the variable `result_params` that was global to the whole web server, I used the [session object provided by Flask](https://flask.palletsprojects.com/en/2.2.x/quickstart/#sessions) to pass the data implicitly in the server while keeping the data linked to specific users of the website.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -16,15 +14,16 @@ def book_input():
         if not text:
             flash('Text is required!')
         else:
-            result_params['title'] = title
-            result_params['text'] = text
+            session['title'] = title
```
Comment (zeegtab) [inline app.py:17]: nit: rename to flask_session or server_session or anything that makes its purpose a bit clearer?

**Review by zeegtab — COMMENTED** [review]
Overall lgtm :)

File: app.py (hunk 1 above)
Comment (BOluokun) [reply app.py:17]: It's not a variable name, it is the object itself: https://github.com/JamesLee-Jones/nlp-timeline-gen/blob/18d64111d14eca3a31189005dff862cb15dd926e/app.py#L1

I could put the line `flask_session = session`, but I feel it's a bit extra.

Comment (zeegtab) [reply app.py:17]: Ah okay my bad! Looks good :)

**Pull request merged**